### ISSUE TITLE
Save update date in metadata instead of using file mod date

### ISSUE THREAD

**Issue by joelmeyerhamme** [JimmXinu/FanFicFare#474]
Copying epub files around causes the file mod date to be changed, which messes with the `-u` switch. I propose to save the update date in the epub metadata, so it can be used to distinguish the last file update that has actually been cause by fanficfare itself. It's possible that the metadata in `content.opf` is already sufficient, however, it doesn't appear to be used, as I get the following warning when running `-u`.

```
FFF: WARNING: 2020-03-11 12:11:56,978: base_writer.py(199): File([...]) Updated(2020-03-11) more recently than Story(2020-03-07) - Skipping 
```

**Comment by JimmXinu** [comment]
Include `always_overwrite: true` in `personal.ini` or by CLI parameter when running `-u` and it will ignore the file `mtime` and only compare the chapter count instead.

(I didn't immediately realize if effected `-u`.  The `always_overwrite` setting is really old; it pre-dates when I started working on the project.  I _know_ it predates the update epub feature.  I always have `always_overwrite: true` in my `personal.ini`, but then I generally only use CLI for testing and use Calibre for my actual reading.  The rest of this I wrote before realizing about `always_overwrite`.)

99.9% of the time if a story is updated, it's because there's a new chapter.  Which is why epub update uses chapter count to determine when an update is appropriate.

The current `<dc:date opf:event="modification">` tag written into epub metadata doesn't include time--timezones are(were?) a significant PITA in python, unfortunately.

Other tools, such as Calibre's Convert and epub2->epub3 feature discard the modification date tag.  And epub3 requires a different date metadata tag that is required to be when the _file_ was updated, not the _content_.

So implementing reading modification from epub looks to me like something isn't needed most of the time, won't be all that precise, and isn't even guaranteed to be there.
